src/pages/Login.js

```diff
@@ -86,7 +101,7 @@ export default () => {
         </ButtonContainer>
         {DB_HACKATHON === 'LHD2021' && <A href="/">Return to Portal</A>}
       </Landing>
-      <ErrorBanner>{error ? <ErrorMessage message={error.message} /> : null}</ErrorBanner>
+      <ErrorBanner>{error ? handleAuthError(error.code, error.message) : null}</ErrorBanner>
```

Comment:
Not sure what the possible states of `error` are, but would
```
{error && <ErrorBanner>{handleAuthError(erro.code, error.message)}</ErrorBanner>
```
be cleaner? Feels weird setting it to `null` otherwise and then setting the visibility of it based on the lack of children. Fine with this though!

Reply:
Ohh for certain errors, we don't want to show the error banner at all, so that's why I made `handleAuthError` decide if we should show the banner